sylt-compiler/src/dependency.rs

```diff
@@ -314,9 +327,10 @@ fn order(
             }
         };
 
-        let (deps, statement) = to_order
-            .get(&global)
-            .expect("Trying to find an identifier that does not exist");
+        let (deps, statement) = to_order.get(&global).expect(&format!(
+            "Trying to find an identifier that does not exist ({:?})",
+            global.0
+        ));
```

Comment:
Yes use being paranoid. If I added this exception - it can happen because I reached it.

Reply:
Do you have an example?

Reply:
Not on me - but I wouldn't have changed it unless it helped me debug something. It has value since it has been hit, it helped me debug and the code costs basically nothing to maintain - feel free to remove it if it becomes cumbersome. But it has payed for itself and will probably give good interest. So I'm keeping it.

Reply:
Sure, I'm just surprised it is even possible, because it should'nt be...